manifests/init.pp

```diff
@@ -19,7 +28,7 @@
   $epel_gpgcheck                          = $epel::params::epel_gpgcheck,
   $epel_repo_gpgcheck                     = $epel::params::epel_repo_gpgcheck,
   $epel_metalink                          = $epel::params::epel_metalink,
-  $epel_managed                           = $epel::params::epel_managed,
+  Boolean $epel_managed                   = true,
```

Comment:
puppet-strings can make use of defaults if they're here and not coming from params.pp